frontend/src/APIClients/SchedulingAPIClient.ts

```diff
@@ -115,7 +115,25 @@ const updateSchedule = async (
     );
     return data;
   } catch (error) {
-    return error as Schedule;
+    return false;
```

Comment:
return false? should the try return true then

Reply:
both will work i think, just needed it for success or error toast. Since the api was returning the updated schedule anyways i kept it like this